ui/app/components/secure-variable-form.js

```diff
@@ -94,19 +85,29 @@ export default class SecureVariableFormComponent extends Component {
     ]);
   }
 
-  @action
-  validatePath(e) {
-    const value = trimPath([e.target.value]);
+  /**
+   * @typedef {Object} DuplicatePathWarning
+   * @property {string} path
+   */
+
+  /**
+   * @type {DuplicatePathWarning}
+   */
+  @computed('args.{model.path,existingVariables}', 'variableNamespace')
```

Comment:
`blocking`: This doesn't need to be a computed. `variableNamespace` is a `tracked` property and the remaining 3 arguments are arguments that are passed to the component that can be a reactive getter.

```
get path() {
   return this.args.model.path;
}

get duplicatePathWarning() {
  // existing logic
}
```

Reply:
`existingVariables` is coming from the controller which peeks the store when the controller is invoked but isn't updating.

Reply:
Good call! Fixed